docs/tuf-spec.txt

```diff
@@ -731,12 +742,27 @@ Version 1.0 (Draft)
            "keyids" : [ KEYID, ... ] ,
            "threshold" : THRESHOLD,
            ("path_hash_prefixes" : [ HEX_DIGEST, ... ] |
-            "paths" : [ PATHPATTERN, ... ])
+            "paths" : [ PATHPATTERN, ... ]),
+           "terminating": TERMINATING,
        }, ... ]
      }
-     
-   ROLENAME is the full name of the delegated role.  For example,
-   "targets/projects"
+
+   ROLENAME is the name of the delegated role.  For example,
+   "projects".
```

Comment:
So does this mean that a ROLENAME *must* only be alphanumeric? How does this new ROLENAME format change how metadata files are placed on the repository? Is this format change backwards-compatible with old clients?